src/include/test/unit_vamana_index.cc

```diff
@@ -325,7 +334,7 @@ TEST_CASE("vamana: small greedy search", "[vamana]") {
   // L = 50, R = 4
   size_t L = 45;
   auto query_id = 14;
-  size_t k = 15;
+  size_t k = 2;
```

Comment:
@lums658 Does this fail with `k = 15`? Curious, why change it?

Reply:
I actually don't know why it was changed.  I changed it back to 15.  There was a typo in one of the expected values, which I fixed.  The test passes.

Reply:
Great, thanks.